tests/unit/s2n_tls13_server_cert_test.c

```diff
@@ -49,14 +49,28 @@ const char tls13_cert_hex[] =
     "1c3b84e0a8b2f759409ba3eac9d91d402dcc0cc8f8"
     "961229ac9187b42b4de10000";
 
+/* certificate chain header. It contains
+   1. Request Context lenght (00)
+   2. Cert chain length (00001b5)
+   3. Cert lenght (0001b0)
+ */
+const char tls13_cert_chain_header_hex[] =
+     "000001b50001b0";
+
 int main(int argc, char **argv)
 {
+    char *tls13_cert_chain_hex;
     BEGIN_TEST();
+    EXPECT_NOT_NULL(tls13_cert_chain_hex = malloc(S2N_MAX_TEST_PEM_SIZE));
```

Comment:
perhaps add a comment that we are creating a blob that concatenate 2 hex strings here?